### DISCUSSION TITLE
[archived] "Installation" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#9549] [General]
# argo-workflows/installation/

https://argoproj.github.io/argo-workflows/installation/

**Comment by melak-cmd** [comment]
since the last version it no longer works on an openshift cluster. I think there are rbac missing...

**Reply by davidkarlsen** [reply]
See https://github.com/argoproj/argo-helm/issues/1559 and linked issues.

**Comment by ognjen-it** [comment]
It looks you have deleted the namespace-install.yaml file from the folder manifests in release 3.4.0.
So, maybe it should be fixed in this documentation.

**Reply by tico24** [reply]
I see it on the current latest release (3.4.3 at the time of writing) and on the 3.4.0 release.
![image](https://user-images.githubusercontent.com/45351296/203826127-d2e2763b-0c9b-4400-91cd-e5d0add691f7.png)

**Reply by ognjen-it** [reply]
You replied 2 months after I wrote this comment, but the documentation was different then :)
Now it's written: `Look for namespace-install.yaml in the release assets`
Previously it was written like this: `kubectl apply -f https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/namespace-install.yaml`
